databend_sqlalchemy/connector.py

```diff
@@ -4,14 +4,12 @@
 #
 # Many docstrings in this file are based on the PEP, which is in the public domain.
 
-from __future__ import absolute_import
-from __future__ import unicode_literals
 import re
 import uuid
 from datetime import datetime
 from databend_sqlalchemy.errors import ServerException, NotSupportedError
 
-from databend_py import Client
+from databend_driver import BlockingDatabendClient
```

Comment:
I think we should keep using `databend-py` and doing more develop things in `databend-py`.